tests/t065_arg_order.py

```diff
@@ -26,8 +26,8 @@ def runcmd(self):
 
         if TestBase.is_32_bit_elf(self):
             # int_mul@arg1 is a 'long long', so we should skip arg2
-            argopt  = '-A "int_mul@arg1/i64" -A "^int_@arg1" '
-            argopt += '-A "int_(add|sub|div)@arg2" -A "int_mul@arg3/x" '
+            argopt  = '-A "int_mul@arg3/x" -A "^int_@arg1" '
+            argopt += '-A "int_(add|sub|div)@arg2" -A "int_mul@arg1/i64" '
             argopt += '-A "int_add@arg1/i32"'
```

Comment:
Then I think the last line `-A int_add@arg1/i32` is not necessary.

Reply:
Thanks. I just moved the line.